### PR TITLE
KOGITO-5447 move tests with non unique FQN

### PR DESCRIPTION
**Pull request by jomarko**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by jomarko** [comment]
This a clone of https://github.com/kiegroup/kogito-examples/pull/775, as we lost changes from that PR during master -> main renaming.

**Comment by jomarko** [comment]
@winklerm @MarianMacik @jstastny-cz sorry to asking you for a review again. It seems the changes introduced by my original PR #775 were lost during the master -> main renaming process.

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@MarianMacik the CI build fail in `process-kafka-multi-springboot` module. Once I try to run that module locally, I get [1]. Any advice how to run given module locally? I do not see any notice about the docker in the readme file.

[1]
```
Caused by: java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$7(DockerClientProviderStrategy.java:215) ~[testcontainers-1.15.3.jar:na]
	at java.base/java.util.Optional.orElseThrow(Optional.java:408) ~[na:na]
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:207) ~[testcontainers-1.15.3.jar:na]
	at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:136) ~[testcontainers-1.15.3.jar:na]
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:178) ~[testcontainers-1.15.3.jar:na]
	at org.testcontainers.LazyDockerClient.getDockerClient(LazyDockerClient.java:14) ~[testcontainers-1.15.3.jar:na]
	at org.testcontainers.LazyDockerClient.authConfig(LazyDockerClient.java:12) ~[testcontainers-1.15.3.jar:na]
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:310) ~[testcontainers-1.15.3.jar:na]
	at org.kie.kogito.testcontainers.KogitoKafkaContainer.start(KogitoKafkaContainer.java:51) ~[kogito-test-utils-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
	at org.kie.kogito.resources.ConditionalSpringBootTestResource.initialize(ConditionalSpringBootTestResource.java:58) ~[kogito-test-utils-2.0.0-SNAPSHOT.jar:2.0.0-SNAPSHOT]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:623) ~[spring-boot-2.3.10.RELEASE.jar:2.3.10.RELEASE]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:367) ~[spring-boot-2.3.10.RELEASE.jar:2.3.10.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:311) ~[spring-boot-2.3.10.RELEASE.jar:2.3.10.RELEASE]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:120) ~[spring-boot-test-2.3.10.RELEASE.jar:2.3.10.RELEASE]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.2.14.RELEASE.jar:5.2.14.RELEASE]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.2.14.RELEASE.jar:5.2.14.RELEASE]
	... 70 common frames omitted
```

**Review by winklerm — APPROVED** [review]
Thanks, looks good to me.

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge.

File: dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kie.dmnpmml.kogito.springboot.example;
+package org.kie.dmn.pmml.kogito.springboot.example;
```
Comment (cristianonicolai) [inline dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java]: @jomarko a general comment, maybe that can be reviewed in another PR, but shouldnt we keep `org.kie.kogito` as the prefix? My suggestion would be to use `org.kie.kogito.dmn.pmml`

Comment (jomarko) [reply dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java]: @cristianonicolai thank you for spotting this. I applied it for **dmn-pmml-springboot** and **dmn-pmml-quarkus** exmaples. If more projects needs the same, I prefer to do it in a separate PR, to limit this PR scope increasing.

Comment (cristianonicolai) [reply dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java]: @jomarko thanks, I can see some remaining `org.kie.dmn.kogito` but all good to do a follow-up PR, thanks for fixing this.

**Review by cristianonicolai — APPROVED** [review]

File: dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java (hunk 1 above)
Comment (jomarko) [reply dmn-pmml-springboot-example/src/test/java/org/kie/dmn/pmml/kogito/springboot/example/ScoreCardTest.java]: yes, I created this task for myself https://issues.redhat.com/browse/KOGITO-5804
